plugins/emojiextender/EmojiExtenderPlugin.php

```diff
@@ -152,43 +158,59 @@ public function emoji_init_handler($sender, $args) {
      * Configure settings page in dashboard.
      *
      * @param SettingsController $sender
-     * @param array $args
      */
-    public function settingsController_emojiExtender_create($sender, $args) {
+    public function settingsController_emojiExtender_create($sender) {
         $sender->permission('Garden.Settings.Manage');
 
-        $cf = new ConfigurationModule($sender);
         $items = [];
-
         foreach ($this->getEmojiSets() as $key => $emojiSet) {
             $manifest = $this->getManifest($emojiSet);
             $selected = '<svg class="icon icon-svg-checkmark" viewBox="0 0 17 17"><use xlink:href="#checkmark" /></svg>';
-            $icon = (isset($manifest['icon'])) ? img($emojiSet['basePath'].'/'.$manifest['icon'], ['alt' => $manifest['name'], 'class' => 'label-selector-image']) : '';
+            $hasManifestIcon = isset($manifest['icon']);
+            $icon = $hasManifestIcon
+                ? img(
+                    $emojiSet['basePath'] . '/' . $manifest['icon'],
+                    [
+                        'alt' => $manifest['name'],
+                        'class' => 'label-selector-image'
+                    ]
+                )
+                : '';
```

Comment:
Can we use `if` statements for such constructions. It seem a bit **big** for ternary operator.

Reply:
This got restructured with the twig view anyways.